spec/acceptance/init_spec.rb

```diff
@@ -2,73 +2,72 @@
 require 'spec_helper_acceptance'
 
 describe 'package task' do
-  include Beaker::TaskHelper::Inventory
-  include BoltSpec::Run
-
   operating_system_fact = os[:family]
   redhat_six = os[:family] == 'redhat' && os[:release].to_i == 6
-
-  def bolt_config
-    { 'modulepath' => RSpec.configuration.module_path }
-  end
-
-  let(:bolt_inventory) { hosts_to_inventory.merge('features' => ['puppet-agent']) }
+  inventory_hash = inventory_hash_from_inventory_file
```

Comment:
inventory_hash should be inside it's own `let` block. I'm not too sure exactly how to work around the fact that you are also writing to an inventory file. You may need to use a combination of a let and some actions in a `before(:all)`.

Reply:
@eimlav Thanks for the vailable comment. If I declare inventory_hash inside `let` block it wont be accessible inside before(:all) to pass to add_feature_to_group or remove_feature_to group or write_to_inventory_file.